src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const basePrice = 40;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const totalAmount = basePrice * days;
+
+  if (days >= LONG_TERM) {
+    return totalAmount - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= SHORT_TERM) {
+    return totalAmount - SHORT_TERM_DISCOUNT;
+  }
+
+  return totalAmount;
```

Comment:
Consider using `const` instead of `let` for the function declaration to prevent reassignment and signal that the function should not be changed.